### PR TITLE
Cosmetic improvements

### PR DESCRIPTION
**Pull request by sohkai**
Adds a number of cosmetic improvements to make the code more cohesive in structure.

**None of the contract behaviours should have been modified**

### CODE REVIEW

File: apps/survey/contracts/Survey.sol [hunk 1]
```diff
@@ -16,31 +16,29 @@ contract Survey is AragonApp {
...
     struct MultiOptionVote {
-        // replacing 2 arrays of ints here by this mapping to save gas
         uint256 optionsCastedLength;
+        // `castedVotes` simulates an array
+        // Each OptionCast in `castedVotes` must be ordered by ascending option IDs
         mapping (uint256 => OptionCast) castedVotes;
     }
 
-    struct SurveyStruct {
+    struct Survey {
```
[the remark this answers is not in the document]
Comment (sohkai) [reply apps/survey/contracts/Survey.sol]: Ahh, good call. Let's leave it as `SurveyStruct` then!

I was wondering why `Vote` worked, but it turns out the contract's name is `Voting`. `Surveying` doesn't sound so good haha.

File: apps/token-manager/contracts/TokenManager.sol [hunk 2]
```diff
@@ -402,9 +410,6 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
     * @return False if the controller does not authorize the approval
     */
     function onApprove(address _owner, address _spender, uint _amount) public returns (bool) {
-        _owner;
-        _spender;
-        _amount;
```
[the remark this answers is not in the document]
Comment (sohkai) [reply apps/token-manager/contracts/TokenManager.sol]: I think if we do, the natspec parser errors out. I think it's OK to leave it with warnings for now.

**Comment by sohkai** [comment]
Pulled the changes to finance to make its uint internals public to https://github.com/aragon/aragon-apps/pull/453

**Comment by bingen** [comment]
Close in favour of #457

**Pull request closed without merging**